package/MDAnalysis/coordinates/TRJ.py

```diff
@@ -529,35 +528,36 @@ def __init__(self, filename, n_atoms=None, mmap=None, **kwargs):
 
         # AMBER NetCDF specs require program and programVersion. Warn users
         # if those attributes do not exist
-        if not (hasattr(self.trjfile, 'program') and
-                hasattr(self.trjfile, 'programVersion')):
+        if not (hasattr(self._f, 'program') and
+                hasattr(self._f, 'programVersion')):
             wmsg = ("NCDF trajectory {0} may not fully adhere to AMBER "
                     "standards as either the `program` or `programVersion` "
                     "attributes are missing".format(self.filename))
             warnings.warn(wmsg)
             logger.warning(wmsg)
 
         try:
-            self.n_atoms = self.trjfile.dimensions['atom']
+            self.n_atoms = self._f.dimensions['atom']
             if n_atoms is not None and n_atoms != self.n_atoms:
                 errmsg = ("Supplied n_atoms ({0}) != natom from ncdf ({1}). "
                           "Note: n_atoms can be None and then the ncdf value "
                           "is used!".format(n_atoms, self.n_atoms))
                 raise ValueError(errmsg)
         except KeyError:
             errmsg = ("NCDF trajectory {0} does not contain atom "
-                      "information".format(self.filename))
+                      "information".format(
+                        self._f.ConventionVersion, self.version))
```

Comment:
I'm not sure why this has been changed - it looks like maybe it was copied from elsewhere by accident?